projects/angular/src/layout/main-container/_variables.header.scss

```diff
@@ -3,47 +3,66 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-@use '../../utils/variables/variables';
+@use '@cds/core/tokens/tokens.scss';
+@use '../../utils/mixins';
 
-// Usage: ../layout/main-containter/_layout.clarity.scss
-// Usage: ../layout/nav/_header.clarity.scss
-// Usage: ../layout/nav/_responsive-nav.clarity.scss
-$clr-header-height: variables.$clr_baselineRem_2_5 !default;
+$clr-header-action-caret-gap: tokens.$cds-global-space-5;
+$clr-header-action-cog-right: calc(
+  var(--clr-header-action-caret-right) + $clr-header-action-caret-gap + var(--clr-header-action-caret-size)
```

Comment:
Same comment as before. This will go away during refactoring and directly substituted with a per-spec token.
When doing these, I was lead by the entities defined in the spec file.
I think these are the ones that should remain exposed as final API.